### PR TITLE
Update docs/dev_guide/intro/mojito_mvc.rst

### PR DESCRIPTION
**Pull request by lazybean**
I think the model function "get_photos" needs the callback function as parameter.

### CODE REVIEW

File: docs/dev_guide/intro/mojito_mvc.rst [hunk 1]
```diff
@@ -97,7 +96,7 @@ instructs Mojito to load the YUI module ``yql`` for getting data.
          this.config = config;        
        },
        // Model function to get data
-       get_photos: function(flickr_query){
+       get_photos: function(callback, flickr_query){
```
Comment (caridy) [inline docs/dev_guide/intro/mojito_mvc.rst]: `callback` should be the second argument instead to follow model pattern.

Comment (lazybean) [inline docs/dev_guide/intro/mojito_mvc.rst]: right, thanks for the review!

**Comment by caridy** [comment]
Thanks for the quick fix @lazybean, I will let @zhouyaoji to review and merge it.

**Comment by zhouyaoji** [comment]
Thanks @lazybean for your pull request.

**Pull request merged**